gocourse05/hw/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type Key struct {
+	x int
+	y int
+}
+
+func initField(field map[Key]string, size int) {
+	fmt.Println("Initing field")
+	for i := 1; i <= size; i++ {
+		s := strconv.Itoa(i)
```

Comment:
I think it would be better to just call printResult at the end of this function. Right now if you want to change the format in which you're printing the field you'd have to do it in two places while this way it could be done only once